app/src/main/java/org/fossasia/openevent/general/auth/SignUpFragment.kt

```diff
@@ -62,6 +64,20 @@ class SignUpFragment : Fragment() {
             redirectToMain()
         })
 
+        rootView.passwordSignUp.addTextChangedListener(object : TextWatcher{
+            override fun afterTextChanged(p0: Editable?) {
+                when {
+                    (passwordSignUp.text.toString().length>=6 || passwordSignUp.text.toString().isEmpty()) -> {
```

Comment:
Second case is redundant. First case already checks that the string can't be empty

Reply:
@iamareebjamal Suppose user writes a password of less than 6 characters (error message appears), now user clear the password field (now because the password field is empty there should not be any error message, this situation is same as if user didn't type any password)

Reply:
@iamareebjamal My second case is if the string is empty then also remove the error message!

Reply:
Yeah alright, my mistake. I didn't see the condition quite clearly :+1: